src/frontend/src/components/anchorInput.ts

```diff
@@ -0,0 +1,84 @@
+import { html } from "lit";
+import { createRef, ref } from "lit/directives/ref.js";
+import { TemplateRef } from "../utils/templateRef";
+
+/** A component for inputting an anchor number */
+export const mkAnchorInput = (
```

Comment:
`mk...` like `mock`?

Reply:
ah no, like `make`! what do you suggest instead?

Reply:
oki doki, all good, thanks

Reply:
clarified offline